inst/NEWS.Rd

```diff
@@ -1,9 +1,14 @@
 \name{NEWS}
 \title{News for Package \pkg{GGIR}}
 \newcommand{\cpkg}{\href{http://CRAN.R-project.org/package=#1}{\pkg{#1}}}
+\section{Changes in version 2.9-4 (GitHub-only-release date:?-?-2023)}{
+  \itemize{
+    \item Part 4: minor bug fixed in the determination of SPT when daysleeper #802
```

Comment:
Was this bug specific to using multiple sleep definitions or does it apply to any analyses of daysleeper data? Researchers who have worked with daysleepers will need to know more detail about the nature of the bug as this could affect their past results. If it affects the sleep estimates then it is not a minor bug but a major bug. However, to assess this people need to understand in which scenario it occured.

Reply:
I got this error even having only one sleep definition (I used all default settings). Just tested with two sleep definitions `timethreshold = c(5, 10)` and `anglethreshold = c(5, 10)` and the error is also there, so unfortunately I think this is affecting always when daysleeper is found in the data. I am happy to test this in other files if you have sample files with frequent daysleeper data (I don't have any).

Reply:
Thanks for confirming, ok maybe  the bug was new as I remember checking the daysleeper functionality extensively around 2020. Either way, the fix seems logical. I have slightly rephrased the changelog now and will it in preparation for a 2.9-4 release in the upcoming hours.